tools/generator/cmd/v2/common/generation.go

```diff
@@ -53,7 +54,7 @@ func (ctx GenerateContext) GenerateForAutomation(readme, repo, specFolderName st
 	for rpName, namespaceNames := range rpMap {
 		for _, namespaceName := range namespaceNames {
 			log.Printf("Process rp: %s, namespace: %s", rpName, namespaceName)
-			singleResult, err := ctx.GenerateForSingleRPNamespace(rpName, namespaceName, "", "", repo, specFolderName)
+			singleResult, err := ctx.GenerateForSingleRPNamespace(rpName, namespaceName, "", "", "", specRPName)
```

Comment:
We can consider to introduce a new struct to wrap these parameters, since they are all string-typed, which might give us difficulties when we want to change this in the future